specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/examples/OutboundNetworkDependenciesEndpointsList.json

```diff
@@ -0,0 +1,242 @@
+{
+  "parameters": {
+    "api-version": "2021-05-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "category": "azure-resource-management",
```

Comment:
>category [](http://example.com/codeflow?start=13&length=8)

Each item in the list response should be modeled like any other ARM resource. This means that it should have an `id`, `name`, and `type` properties. Any RP-specific properties should be included in the `properties` property. (Yes, this is required even though this is a proxy resource.)

Reply:
Hi @majastrz ,
This is already answered and signed off in https://github.com/Azure/azure-rest-api-specs/pull/14332#discussion_r629606824 when merging the PR to dev branch

Reply:
Given that you're ok with the suboptimal API design, the restrictions it causes (as explained by @pilor in the linked comment), and the fact that it was already approved by API review, I'll let it slide.

Reply:
Also keep in mind that this is a one-off exception and it should not be used to justify similar designs elsewhere in the RP.